wp-content/themes/borderzine/inc/widgets/class-borderzine-6-col-widget.php

```diff
@@ -0,0 +1,301 @@
+<?php
+
+/**
+ * Borderzine 6-Column recent posts widget and associated functions
+ *
+ * Copied from Largo Recent Posts
+ */
+class Borderzine_6_Col_Widget extends WP_Widget {
+
+	/**
+	 * Register widget with WordPress.
+	 */
+	function __construct() {
+
+		$widget_ops = array(
+			'classname' => 'borderzine-6-col-widget',
+			'description' => __( 'A flexible widget to display recent photo gallery posts.', 'borderzine' )
+		);
+		parent::__construct(
+			'borderzine-6-col-widget', // Base ID
+			__( 'Borderzine 6 Column Widget', 'borderzine' ), // Name
```

Comment:
I'll copy this naming scheme over to https://github.com/INN/umbrella-borderzine/pull/67